tests/test_miscellaneous.py

```diff
@@ -75,6 +75,34 @@ def check_parent(child, parent):
     def test_str(self):
         assert_equal(str(Granularity.SOFTWARE), 'SOFTWARE')
 
+    def test_comparisons(self):
+        assert_equal(Granularity.SOFTWARE, Granularity.SOFTWARE)
+        assert_equal(Granularity.PACKAGE, Granularity.PACKAGE)
+        assert_equal(Granularity.CLASS, Granularity.CLASS)
+        assert_equal(Granularity.METHOD, Granularity.METHOD)
+        assert_equal(Granularity.FUNCTION, Granularity.FUNCTION)
+
+        assert_true(Granularity.SOFTWARE > Granularity.PACKAGE)
```

Comment:
I think we can use `assert_greater` here to get some nicer error messages in case of failure (same for the next comparison block afterwards. It would be even better to use [test case generators](http://nose.readthedocs.org/en/latest/writing_tests.html#test-generators) so each failure can be reported individually: